app/views/api/v2/auth_source_internals/show.json.rabl

```diff
@@ -0,0 +1,7 @@
+object @auth_source_internal
+
+extends "api/v2/auth_source_internals/main"
+
+node do |auth_source_internal|
+  partial("api/v2/taxonomies/children_nodes", :object => auth_source_internal)
```

Comment:
internal don't seem to support taxonomies, this results in {} sent for both org and loc ids